src/lib.rs

```diff
@@ -119,7 +119,7 @@
 
 use std::fmt;
 
-use heck::SnakeCase;
+use heck::ToSnakeCase as _;
```

Comment:
Why the `as _`?

Reply:
This emphasizes that the imported name is not used directly, so doesn't use its name in the current namespace, but rather only brings the trait implementation to the scope.

Reply:
Ah I see. I'm not a fan because it reminds me of stateful imports and it's pattern in languages like Go and Python. Either way, it's just a nitpick so I'll be passing this forward